program-runtime/src/sysvar_cache.rs

```diff
@@ -210,6 +210,62 @@ impl SysvarCache {
     pub fn reset(&mut self) {
         *self = SysvarCache::default();
     }
+
+    /// [Syscall Sol-Get-Sysvar]: This is a mockup implementation.
```

Comment:
@2501babe I left this silly thing in for now, and figured your Sysvar Cache overhaul could just replace it altogether.

The rest of the PR is useful though!